editoast/src/views/paced_train.rs

```diff
@@ -0,0 +1,128 @@
+use crate::error::Result;
+use axum::extract::Json;
+use axum::extract::State;
+use axum::{response::IntoResponse, Extension};
+use editoast_authz::BuiltinRole;
+use editoast_derive::EditoastError;
+use editoast_models::DbConnectionPoolV2;
+use editoast_schemas::{paced_train::PacedTrainBase, train_schedule::TrainScheduleBase};
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+use utoipa::ToSchema;
+
+use super::AuthenticationExt;
+use crate::views::train_schedule::BatchRequest;
+use crate::{
+    models::paced_train::{PacedTrain, PacedTrainChangeset},
+    views::AuthorizationError,
+};
+
+crate::routes! {
+    "/paced_train" => {
+        delete,
+    },
+}
+
+editoast_common::schemas! {
+    PacedTrainBase,
+}
+
+#[derive(Debug, Error, EditoastError)]
+#[editoast_error(base_id = "paced_train")]
+pub enum PacedTrainError {
+    #[error("{number} paced train(s) could not be found")]
+    #[editoast_error(status = 404)]
+    BatchPacedTrainNotFound { number: usize },
```

Comment:
```suggestion
    BatchPacedTrainNotFound { count: usize },
```

nit: "number" may be mistaken for an ID